programs/install/Install.cpp

```diff
@@ -1083,7 +1081,7 @@ namespace
             sleepForSeconds(1);
         }
 
-        if (try_num == num_tries)
+        if (try_num == timeout)
```

Comment:
Ok.

The semantics of this code is slightly incorrect.

It is performing `timeout` number of tries, each of them doing some check and sleeping for one second. With a large number of tries, it does not correspond to waiting for the timeout.

Reply:
Let's rename the config parameter and the variable to `max-tries`.

Reply:
Yeah, I also thought about this, decided that making it true timeout does not worth it for such a small binary, but haven't thought about different name for the option. Thanks for suggestion. Applied.